docs/shared/router-common-config.mdx

```diff
@@ -0,0 +1,210 @@
+```yaml
+# Common values. Overridden by in-house configurations.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Anything we set here applies to the in-house-router chart, and 
+# all configuration that we set here is pushed to all sub-charts
+# (because of the way Helm works:
+#  https://helm.sh/docs/chart_template_guide/subcharts_and_globals/)
+
+# Set fullnameOverride here for virtualservice to use
+fullnameOverride: router
+
+router:
+  serviceAccount:
+    # We do that ourselves in permissions.yaml
+    create: false
+    name: router
+  fullnameOverride: router
+  router:
+    args: [--hot-reload]
+    configuration:
+      limits:
+        experimental_http_max_request_bytes: 20000000 #20MB
+      traffic_shaping:
+        router:
+          timeout: 6min
+        all:
+          experimental_enable_http2: false # to disable for all subgraphs
+          timeout: 5min
+      supergraph:
+        introspection: true
+        # We are intentionally DISABLING DEFER SUPPORT in the Router until
+        # we are out of our transition period from the Gateway to the Router.
+        defer_support: false
+        listen: 0.0.0.0:80
+        path: /graphql
+      # We keep subgraph errors on in our own environment, always.
```

Comment:
Is it worth giving the rationale for this, or just to say:

```suggestion
      # We recommend keeping subgraph errors on in your own environment, always.
```

Reply:
We should delete that comment. We are keeping subgraph errors on in-house to provide backwards compatibility with existing Gateway behaviour. It's not ideal, since it may leak implementation detail or PII(Personal Identifiable Information), but it's a requirement of our particular deployment.